.nvmrc

```
@@ -1 +1 @@
-16.10.0
+18.17.1
```

Should we do `18.17` so we are on latest patch? Or is that too unstable for the JS ecosystem.

---

Yep, that's fine by me. 18.17.1 is the current latest patch anyway.

We could even use just `18`, but I think it was some dot release of 18 that made NPM 9 the default, and I don't feel like looking up which one that was

```suggestion
18.17
```